setup.py

```diff
@@ -13,7 +13,9 @@
     author="Trail of Bits",
     version="0.3.4",
     packages=find_packages(),
-    python_requires=">=3.8",
+    # Python 3.12.0 on Windows suffers from https://github.com/python/cpython/issues/109590
+    # breaking some of our integrations. The issue is fixed in 3.12.1
+    python_requires=">=3.8,!=3.12.0",
```

Comment:
Should be treat this as breaking?

Reply:
we can drop this restriction if you feel it's too much. I tried looking to see if there was a way to specify the version exception just for Windows but did not find anything suitable.

Reply:
AFAICT pip should handle this and not leave anyone using python3.12.0 and crytic-compile with a broken build